src/converters/date.js

```diff
@@ -3,7 +3,7 @@ import { isNil } from "lodash";
 
 export default function(value) {
   if(!isNil(value) && value !== "") {
-    let temp = Moment.unix(value).utc();
```

Comment:
Won't this mean that the data being passed back to the server will also be in the user's timezone?

Reply:
Yes it would. But I've decided not to submit this PR, at least for now. I've been experimenting with this idea of using the user's local timezone because we've encountered several situations where converting a date (day only) to UTC changed the calendar day that was saved because of the difference between UTC and the user's timezone. But sending the user's timezone time creates a different set of issues too. So I think we'll stick with UTC and make the corrective adjustments where needed on the server.